Userland/Libraries/LibDNS/Answer.cpp

```diff
@@ -24,7 +24,7 @@ Answer::Answer(Name const& name, RecordType type, RecordClass class_code, u32 tt
 
 bool Answer::has_expired() const
 {
-    return time(nullptr) >= m_received_time + m_ttl;
+    return time(nullptr) >= static_cast<time_t>(m_received_time + m_ttl);
```

Comment:
Does the 64-bit Hurd port have signed time_t? That seems a very odd choice for such a type.

Reply:
* I haven't tried to build this on 64-bit Hurd, only on 32-bit. So maybe this commit should have gone into the 32-bit fixes branch actually.
* Yes, `time_t` is always signed in glibc. This is documented [here](https://www.gnu.org/software/libc/manual/html_node/Time-Types.html).

    If `time_t` was unsigned, the 2038 problem would have been the 2106 problem or something like that.